app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt

```diff
@@ -1396,20 +1854,25 @@ class AppointmentRepositoryAndroidTest {
     }
 
     fun toOverdueAppointment(): OverdueAppointment {
-      checkNotNull(bloodPressureMeasurement) {
-        "Cannot create an Overdue Appointment without a Blood Pressure Measurement"
+      if (bloodPressureMeasurement == null && bloodSugarMeasurement == null) {
+        throw AssertionError("Cannot create an Overdue Appointment without a Blood Pressure Measurement and Blood Sugar Measurement")
```

Comment:
"Blood Pressure Measurement **and** Blood Sugar Measurement" implies that we need at least one of each when in reality, we need one of either. Can we change the assertion message to reflect this?